setup.py

```diff
@@ -43,7 +43,8 @@
     # update with a tag from the nawi-hub/idaes-pse
     # when a version of IDAES newer than the latest stable release from PyPI
     # will become needed for the watertap development
-    "idaes-pse[prerelease] @ https://github.com/watertap-org/idaes-pse/archive/1.12.1.watertap.2022.02.04.zip",
+    #"idaes-pse[prerelease] @ https://github.com/watertap-org/idaes-pse/archive/1.12.1.watertap.2022.02.04.zip",
+    "idaes-pse[prerelease] @ git+https://github.com/andrewlee94/idaes-pse.git@costing_2"
```

Comment:
```suggestion
    "idaes-pse[prerelease] @ https://github.com/watertap-org/idaes-pse/archive/1.14.0dev0.watertap.2022.03.11.zip"
```

I've just created the tag on watertap-org/idaes-pse: https://github.com/watertap-org/idaes-pse/tree/1.14.0dev0.watertap.2022.03.11